TestAdapterforGoogle.TestDev17.yml

```diff
@@ -129,6 +136,11 @@ extends:
           clean: true
           fetchDepth: 1
           persistCredentials: true
+        - checkout: VCLS-Extensions
+          displayName: 'Checkout VCLS-Extensions ADO Repo to copy public key'
+          clean: true
+          fetchDepth: 1
```

Comment:
Just note that this is already the default, according to the docs:
https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/steps-checkout?view=azure-pipelines#shallow-fetch

Reply:
The GH UI does not highlight the line I selected (o.O).
I was talking about the `fetchDepth` field.